.github/workflows/ci.yml

```diff
@@ -72,6 +88,14 @@ jobs:
     name: Test Cheatnet
     runs-on: ubuntu-latest
     steps:
+      - name: Set NODE_URL for foundry-rs/starknet-foundry
+        if: ${{ github.event.pull_request.head.repo.full_name == 'foundry-rs/starknet-foundry' }}
+        run: echo "NODE_URL=$(echo ${{ secrets.NODE_URL }}:7070)" >> $GITHUB_ENV
+
+      - name: Set NODE_URL for forks
+        if: ${{ github.event.pull_request.head.repo.full_name != 'foundry-rs/starknet-foundry' }}
+        run: echo "NODE_URL=$(echo ${{ secrets.NODE_URL }})" >> $GITHUB_ENV
+
```

Comment:
Not really a fan of that solution, are we sure the last url path segment will be the rpc version in whole ecosystem? Also hardcoing the port here doesn't seem good, imo it would be better to hardcode it in the variable

Reply:
As far as I know, all ([Juno](https://juno.nethermind.io/), [Pathfinder](https://github.com/eqlabs/pathfinder?tab=readme-ov-file#json-rpc-api), [Papyrus](https://github.com/starkware-libs/papyrus/?tab=readme-ov-file#sending-api-requests-to-the-node)) node providers agreed to use the same suffix.

As for hardcoding the port (as we discussed offline), it is for the possibility of using out other node with a different port (so that the `NODE_URL` variable stays the same).

Reply:
This one i'm fine with, but the latter i'd rather have in 2 variables (since it's a different variable essentially)

Reply:
I'm not sure we should care about setting this for forks. From my experience very few (if any) people actually run CIs on their forks, they just run tests locally and then run tests on our CI when we approve it. If someone is savvy enough to set all the necessary envs and enable actions on their fork, then they are savvy enough to change this port on their fork imo. 

As a side note, I quite like the flow we have in sncast, where we set the whole url in env, but I already asked a question about it in https://github.com/foundry-rs/starknet-foundry/pull/2003/files#r1572097275, since I'm not sure this is applicable to snforge in the same degree it is to sncast.

Reply:
All right, so I guess we remove `NODE_URL` env var setting for forks?